Ray.jl/docs/Project.toml

```diff
@@ -0,0 +1,6 @@
+[deps]
```

Comment:
isn't there normally a Manifest for the docs as well?

Reply:
I just nuked all the docs stuff sicne it doesn't make sense trying to build them right now (all the caching etc. would have to be set up in teh docs build)